cmd/escrow/config.toml

```diff
@@ -0,0 +1,9 @@
+[storage]
```

Comment:
I'd actually prefer converting everything to toml, I just didn't want to do it in this PR. I'll file an issue for it. Viper will make it little more than just rewriting the json files and updating our Dockerfiles and docker-compose files to point at the toml versions.